tools/common/Driver.cs

```diff
@@ -202,6 +261,56 @@ static bool ParseOptions (Application app, Mono.Options.OptionSet options, strin
 					app.EnvironmentVariables.Add (name, value);
 				}
 			);
+			options.Add ("registrar:", "Specify the registrar to use (dynamic, static or default (dynamic in the simulator, static on device))", v => {
+				var split = v.Split ('=');
+				var name = split [0];
+				var value = split.Length > 1 ? split [1] : string.Empty;
```

Comment:
Do we trust people not doing 'stupid' things like Static etc..?

Reply:
They'll just get an error saying to use `static` instead.